static/js/calculate_score.js

```diff
@@ -359,6 +391,141 @@ var calculatePolyScore = async () => {
     }
 }
 
+/**
+ * 
+ * @param {*} gwasUploadFile the uploaded tab separated GWAS data file
+ * @param {*} gwasRefGen the reference genome of the GWAS upload data
+ * @param {*} refGen the reference genome of the samples
+ * @returns the associations data object used in calculating polygenic risk scores
+ */
+async function getGWASuploadData(gwasUploadFile, gwasRefGen, refGen) {
+    var fileContents = await readFile(gwasUploadFile);
+    fileLines = fileContents.split("\n")
+
+    //TODO? put in that they have the option to include other columns and we will add that to the output??
+
+    associationsDict = {}
+    chromSnpDict = {}
+    studyIDsToMetaData = {}
+
+    sii = -1 //studyID index
```

Comment:
Now that you've figured out the logic for this next part, I would rename these variables so they are easier to read (ex: studyIDIndex)

Reply:
I opted to add more comments at each point instead of renaming the variables. I like how short the variable names are now, since they are just index numbers. Making the variable names longer would increase the code line length